src/test/java/bb/example/UserInterfaceTest.java

```diff
@@ -0,0 +1,245 @@
+package bb.example;
+
+import bb.example.services.IOHandler.InputHandler;
+import bb.example.services.IOHandler.OutputHandler;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.mockito.Mockito.*;
+
+
+@ExtendWith(MockitoExtension.class)
+class UserInterfaceTest {
+    @Mock
+    InputHandler inputHandler;
+    @Mock
+    OutputHandler outputHandler;
+    UserInterface userInterface;
+
+    @BeforeEach
+    void setUp() {
+        userInterface = new UserInterface(new PersonDatabase(), inputHandler, outputHandler);
+    }
+
+    @Test
+    @DisplayName("Should return \"Enter number from 1 to 7\" when user input is not number from 1 to 7 in menu")
+    void shouldReturnMustBeNumberInRangeWhenInputIsOutOfBoundInMenuTest() {
+        when(inputHandler.getInput())
+                .thenReturn("0", "8", "7");
+        userInterface.menuSelectionLoop(inputHandler);
+
+        verify(outputHandler, times(2)).print("Enter number from 1 to 7");
+    }
+
+    @Test
+    @DisplayName("Should return \"Invalid number. Please enter number.\" when user input is not number")
+    void shouldReturnInvalidNumberWhenInputIsLetterInMenuTest() {
+        when(inputHandler.getInput())
+                .thenReturn("input", " ", "", "7");
+        userInterface.menuSelectionLoop(inputHandler);
+
+        verify(outputHandler, times(3)).print("Invalid number. Please enter number.");
+    }
+
+    @Test
+    @DisplayName("Should return String of successful added person when user input is 1")
+    void shouldReturnAddedPersonWhenInputOneAndPersonGivenTest() {
```

Comment:
All these tests only verify the output, but maybe it would make sense to verify that a person with given attributes has actually been created in the database? Similarly for other tests.